### DISCUSSION TITLE
SharePoint Add-In retirement in Microsoft 365 is impacting PnP modern search solution?

### DISCUSSION THREAD

**Discussion by itebet** [microsoft-search/pnp-modern-search#3695] [General]
Dear PnP modern search team, 
Can you please clarify if your solution is going to be impacted by the "SharePoint Add-In retirement in Microsoft 365" that Microsoft is planning on July 1st 2024? 

https://learn.microsoft.com/en-gb/sharepoint/dev/sp-add-ins/retirement-announcement-for-add-ins 

I just wondering if it won't possible to upgrade anymore the solution from that date.

**Comment by kasperbolarsen** [comment]
Hi, please see the FAQ : https://microsoft-search.github.io/pnp-modern-search/QnA/

**Reply by itebet** [reply]
Thank you, that's a clear answer :-)
